src/main/java/androidx/view/EditText.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.view
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.EditText
+
+/**
+ * Set and Get the string value of EditText content
+ */
+var EditText.content
+    get() = text.toString()
+    set(value) {
+        setText(value)
+    }
+
+/**
+ * @returns Weather the EditText text is empty or not
```

Comment:
Do you mean "Weather"?

Reply:
Rain or shine, the EditText is empty or not.

Reply:
Haha 😂